webv2/uniqueid/uniqueid_assignment.go

```diff
@@ -258,3 +258,50 @@ func InitObjectId() {
 	sessionState := session.GetSessionState()
 	sessionState.Counter.ObjectId = "0"
 }
+
+//CopyUniqueIdToSpannerTable copy ids from source table to spanner table content
+func CopyUniqueIdToSpannerTable(conv *internal.Conv, spannertablename string) {
+	sourcetablename := conv.ToSource[spannertablename].Name
+	spannertable := conv.SpSchema[spannertablename]
+	sourcetable := conv.SrcSchema[sourcetablename]
+	spannertable.Id = sourcetable.Id
+
+	for spannercolumnname, spannercolumn := range spannertable.ColDefs {
+		if spannercolumn.Name == "synth_id" {
+			columnuniqueid := GenerateColumnId()
+			spannercolumn.Id = columnuniqueid
+			conv.SpSchema[spannertablename].ColDefs[spannercolumnname] = spannercolumn
+		}
+	}
+
+	for _, sourcecolumn := range sourcetable.ColDefs {
+		for spannercolumnname, spannercolumn := range spannertable.ColDefs {
+			if sourcecolumn.Name == spannercolumn.Name {
+				spannercolumn.Id = sourcecolumn.Id
+				conv.SpSchema[spannertablename].ColDefs[spannercolumnname] = spannercolumn
```

Comment:
since at the end we are setting `conv.SpSchema[spannertablename] = spannertable` we can replace `conv.SpSchema[spannertablename]` with `spannertable` everywhere

Reply:
Replaced